### ISSUE TITLE
Callsigns not recognized: ID UNKNOWN

### ISSUE THREAD

**Issue by sergev** [sergev/dmrconfig#3]
After contacts.CSV database has been updated with "dmrconfig -u", the radio cannot recognize an incoming callsign and shows "ID UNKNOWN".

**Comment by sergev** [comment]
Region 0x200003-0x204002 of configuration memory contains a search helper for the ContactsCSV table. The helper is organized as a list of pairs <idbase, index>. 
- idbase - high bits 23:12 of DMR ID
- index - number in ContactsCSV table, starting from 1 (20 bits)

For example:
```
              id[23:12]
       Ncontacts  |   index[19:0]
        vvvvvvvv //\  /||\\
200000  00 40 00 5a d0 00 01 5a e0 10 00 5a f0 20 00 5b  .@.Z...Z...Z. .[
200010  00 30 00 5b 10 40 00 ff ff ff ff ff ff ff ff ff  .0.[.@..........
200020  ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff  ................
*
204000  ff ff ff 
```
Need to modify dmrconfig to generate a valid search helper.

**Comment by sergev** [comment]
Fixed in ae79587ff9c54c3172c14d5f6e3e81100b80195e: a search index is created for the ContactsCSV table.
